node-require/calculate.js

```diff
@@ -0,0 +1,16 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+var firstNum = parseInt(process.argv[2]);
+var secondNum = parseInt(process.argv[4]);
```

Comment:
This should produce the right output for non-whole numbers as well.

Reply:
@TimelikeClosure , I've updated it to take non-whole numbers. Please review! thanks